.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
```

Comment:
Hmm, I've never seen lint config files with a js extension. What's the reasoning behind this?